bindings/java/iota-wallet-java/examples/src/main/CallbackEvents.java

```diff
@@ -0,0 +1,87 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import org.iota.Wallet;
+import org.iota.api.NativeApi;
+import org.iota.types.*;
+import org.iota.types.ClientConfig.ApiTimeout;
+import org.iota.types.account_methods.GenerateAddresses;
+import org.iota.types.account_methods.SyncAccount;
+import org.iota.types.events.Event;
+import org.iota.types.events.EventListener;
+import org.iota.types.events.transactionprogress.TransactionProgressEvent;
+import org.iota.types.events.wallet.WalletEventType;
+import org.iota.types.exceptions.WalletException;
+import org.iota.types.ids.account.AccountAlias;
+import org.iota.types.secret.StrongholdSecretManager;
+
+public class CallbackEvents {
+
+    static class Callback implements EventListener {
+        @Override
+        public void receive(Event event) {
+            System.out.println("All events receive: ");
+            System.out.println(event.getEvent());
+        }
+    }
+
+    public static void main(String[] args) throws WalletException, InterruptedException {
+        // This example assumes that a wallet has already been created using the
+        // ´CreateWallet.java´ example.
+        // If you have not run the ´CreateAccount.java´ example yet, run it first to
+        // ensure that the wallet can be loaded correctly.
+        Wallet wallet = new Wallet(new WalletConfig()
+                .withClientOptions(
+                        new ClientConfig().withApiTimeout(new ApiTimeout().withSecs(60)).withNodes(Env.NODE))
+                .withSecretManager(
+                        new StrongholdSecretManager(Env.PASSWORD, null, Env.SNAPSHOT_PATH))
+                .withCoinType(CoinType.Shimmer));
+
+        wallet.listen(new EventListener() {
+
+            @Override
+            public void receive(Event event) {
+                System.out.println(
+                        System.lineSeparator() + "Event receive: " + event.getEvent().getClass().getSimpleName());
+                if (event.getEvent() instanceof TransactionProgressEvent) {
+                    TransactionProgressEvent progress = (TransactionProgressEvent) event.getEvent();
+                    System.out.println(progress.toString());
+                } else {
+                    System.out.println(event.getEvent().toString());
+                }
+            }
+
+        }, WalletEventType.values());
```

Comment:
WalletEventType.values()) says that it should listen to all events here right? Maybe add a comment